baker/siteRenderers.tsx

```diff
@@ -341,7 +342,20 @@ export const renderNotFoundPage = () =>
 
 export async function makeAtomFeed() {
     const posts = (await getBlogIndex()).slice(0, 10)
+    return makeAtomFeedFromPosts(posts)
+}
+
+// We don't want to include topic pages in the atom feed that is being consumed
+// by Mailchimp for sending the "immediate update" newsletter. Instead topic
+// pages announcemements are sent out manually.
```

Comment:
good point. I don't know if changing the feed URL resets the pointer or if it is tied to the campaign instead. I'll ask.

Reply:
mailchimp response: it won't be sent.

As a security, we should make the change 24h after the last issue. Even if the campaign is reset, nothing would go:

> The first time your RSS email sends, we'll only include posts from the last 24 hours for daily, from the last seven days for weekly, and the last 30 days for monthly. After that, each new send will pull all posts made to your blog feed since the last send.

https://mailchimp.com/en/help/share-your-blog-posts-with-mailchimp/